plugins/omazureeventhubs/omazureeventhubs.c

```diff
@@ -712,12 +747,30 @@ CODESTARTcommitTransaction
 			}
 		}
 
-		if (	pWrkrData->bIsConnected == 1 &&
-			iNeedSubmission > 0 &&
-			pn_link_credit(pWrkrData->pnSender) > 0) {
-			pn_connection_wake(pWrkrData->pnConn);
-			/* Grant enough credit to bring it up to BATCH: */
-			// pn_link_flow(pWrkrData->pnSender, iToSubmit);
```

Comment:
pls remove commented-out code

Reply:
Sure, removed that commented code.

Reply:
I still see it?

Reply:
I see it removed in the force push compare: 
https://github.com/rsyslog/rsyslog/compare/4f01cf92ce5c1927bb8412c9637ce79a83914fed..eece807639fa8e3df62808394f847792417f6cf9

Reply:
sry, my fault. this is **removed** code...